### ISSUE TITLE
Broken Nvidia GPU acceleration in ROS 2 Superthin Templates

### ISSUE THREAD

**Issue by pawanw17** [JdeRobot/RoboticsAcademy#2516]
When trying to run an acceleration with nvidia GPU acceleration, it shows a black screen
```
docker run --rm -it --gpus all --device /dev/dri -e DRI_NAME=card1 -p 7164:7164 -p 6080:6080 -p 1108:1108 -p 7163:7163 jderobot/robotics-academy:4.5.11
``` 

![image](https://github.com/JdeRobot/RoboticsAcademy/assets/40732709/9a8a52e6-7a44-42af-afca-d028847e1ed8)

**Comment by Syekeon** [comment]
it's strange, I use the same line (with card0 in my case) and it's working good with my Nvidia GPU.
![image](https://github.com/JdeRobot/RoboticsAcademy/assets/117100820/f9189b63-2641-41d7-8577-1ab8bfe5c2ef)

**Comment by jmplaza** [comment]
0.95 is definitely a good Gazebo RTF (Real Time Factor) :-)
Which version of docker itself are you using? and which Operating System?

Could you please @Syekeon check with htop the % of CPU used by gzserver and gzclient processes?

**Comment by pawanw17** [comment]
Hi @Syekeon for me card0 works too, but those are intel integrated graphics, it fails on card1(Nvidia)

**Comment by Syekeon** [comment]
@pawanw17 in my case, I don't have integrated graphics so my nvidia card is in card0. Maybe is a issue when using multiple gpu cards?

@jmplaza I tested in Windows 11 with 26.0.0 Docker version. I got 0.90 RTF in Follow Line on average. I checked the % of CPU use with htop and I looked that it's consuming a high power of CPU when the exercise is running.

![image](https://github.com/JdeRobot/RoboticsAcademy/assets/117100820/35fc181b-b103-42cb-aad5-b80094d59070)
